### ISSUE TITLE
Can not track UITextField UIControlEventEditingDidEnd event on iOS12.

### ISSUE THREAD

**Issue by SecondDog** [mixpanel/mixpanel-iphone#800]
The root cause is that in iOS12 UITextField method didMoveToWindow was **overwrite and it did not call its super method。** So when mixpanel try to hook UIControl Class's didMoveToWindow Method，the UITextField‘s didMoveToWindow was not hooked and called .That result in the event not actually binded and tracked.

Fix it like this:

in **UIControlBinding.m**
 initWithEventID:onPath:withControlEvent:andVerifyEvent:index:navTitle

add code:

```
......
if (controlEvent == UIControlEventEditingDidEnd ){
            [self setSwizzleClass:[UITextField class]];
}
......
```

**Comment by zihejia** [comment]
Thanks so much @SecondDog, do you mind making a pull request for this fix?

**Comment by SecondDog** [comment]
> Thanks so much @SecondDog, do you mind making a pull request for this fix?

OK 。 No Problem。

**Comment by zihejia** [comment]
We have released v3.3.4 and has a fix for this. Thanks @SecondDog !
